Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
+
+In this guide we'll make heavy use of an awesome best practices collection from the [PLOS | Public Library of Science](https://www.plos.org/). 
```

Comment:
We will make ....

(remove _In this guide_ -- its redundant)

Reply:
> ....use of an awesome best practices collection....

...use of a collection of awesome best practices...  --- reads better

Reply:
👍